src/Base/Parameter.cpp

```diff
@@ -1526,34 +1526,14 @@ DOMPrintFilter::FilterAction DOMPrintFilter::acceptNode(const DOMNode* node) con
     }
 
     switch (node->getNodeType()) {
-    case DOMNode::ELEMENT_NODE: {
-        return DOMNodeFilter::FILTER_ACCEPT;
-
-        break;
-    }
-    case DOMNode::COMMENT_NODE: {
-        return DOMNodeFilter::FILTER_ACCEPT;
-        break;
-    }
-    case DOMNode::TEXT_NODE: {
-        return DOMNodeFilter::FILTER_ACCEPT;
-        break;
-    }
-    case DOMNode::DOCUMENT_TYPE_NODE: {
-        return DOMNodeFilter::FILTER_REJECT;  // no effect
-        break;
-    }
-    case DOMNode::DOCUMENT_NODE: {
-        return DOMNodeFilter::FILTER_REJECT;  // no effect
-        break;
-    }
-    default : {
-        return DOMNodeFilter::FILTER_ACCEPT;
-        break;
-    }
+        case DOMNode::DOCUMENT_TYPE_NODE:
+        case DOMNode::DOCUMENT_NODE: {
+            return DOMNodeFilter::FILTER_REJECT;// no effect
+        }
+        default: {
+            return DOMNodeFilter::FILTER_ACCEPT;
+        }
     }

```

Comment:
I was thinking to something like that :
```suggestion
        default: {
            // Let return function default value
        }
    }
    return DOMNodeFilter::FILTER_ACCEPT;
```

Reply:
But why create an empty default and then separately a return? What is gained by the extra code?

Reply:
It's clearer what is the function default return.
And if one want to add a new case like 
```
case foo:
    if (bar):
        break;
    else:
       foobar();
    [[fallthrough]]
```
That directly works.